.circleci/config.yml

```diff
@@ -386,6 +395,86 @@ jobs:
             mv ~/js_types/* ~/dist/
             ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${CIRCLE_TAG} ~/dist/
 
+  publish-cocoapod:
+    macos:
+      xcode: '10.1.0'
+    steps:
+      - run:
+          name: update podspec
+          working_directory: release
+          command: |
+            VERSION=$(echo $CIRCLE_SHA1 | cut -c -7)
+            if [ "${CIRCLE_TAG}" != "" ]; then
+                VERSION=${CIRCLE_TAG}
+                VERSION=$(echo $VERSION | cut -c 2-)
+            fi
+            sed -i.bak 's/<version>/${VERSION}/g' TextileCore.podspec
+            pod trunk push TextileCore.podspec --allow-warnings
```

Comment:
This is an easy one, just publishes the Pod which references the files in the GH release. Uses the `COCOAPODS_TRUNK_TOKEN` env var by default